src/config/dynamic-theme-fixes.config

```diff
@@ -6206,17 +6206,61 @@ gog.com
 
 INVERT
 i.icn.icn--close
+svg.icon-wrapper-icon
+svg.big-spot__add-to-cart-icon
+svg.ic-svg:not(.button__icon):not(.productcard-slider__nav-icon)
+.productcard-thumbnails-slider-pagination
+.carousel-pagination__page
+.big-spot__carousel-pages-container
+.big-spots__arrow-icon
+.discover-games-more__icon
```

Comment:
I will also have a look at these elements as these are hitting a case that dark reader _rightfully_ doesn't handle yet, but in this specific case it can be handled correctly.